### PR TITLE
File open

### PR DESCRIPTION
**Pull request by harshitks2203**
Completed FileOpen

### CODE REVIEW

File: app/fileMenu.js [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
 };
-let saveFlag;
+let saveFlag, openFlag;
 const getSavePath = () => savePath;
 const setSavePath = path => {
   savePath = path;
 };
 
+const getOpenPath = () => openPath;
+const setOpenPath = path => {
```
Comment (Jay9596) [inline app/fileMenu.js]: What is the reason for `setOpenPath`?
Shouldn't it just `setSavePath` the path from where the file is opened?

File: app/fileMenu.js [hunk 2]
```diff
@@ -1,17 +1,28 @@
+const EDITOR = require("./editor");
```
[the remark this answers is not in the document]
Comment (Jay9596) [reply app/fileMenu.js]: You won't need to import EDITOR, all values you used are global to the application window.

[the remark this answers is not in the document]
Comment (harshitks2203) [reply app/fileMenu.js]: ok

File: app/fileMenu.js (hunk 1 above)
Comment (harshitks2203) [reply app/fileMenu.js]: i thought that we have to do that when we are dealing with opening and saving file. As previously did for saving.

**Pull request merged**